src/servers/ZoneServer2016/handlers/commands/commands.ts

```diff
@@ -3397,7 +3397,43 @@ export const commands: Array<Command> = [
       console.log(server.weatherManager.weather);
       server.weatherManager.sendUpdateToAll(server, client, false);
     }
-  }
+  },
+  {
+    name: "stats",
+    permissionLevel: PermissionLevels.DEFAULT,
+    keepCase: true,
+    execute: async (server: ZoneServer2016, client: Client) => {
 
+      const characterData = await server._db
+        .collection("characters")
+        .findOne({ characterId: client.character.characterId });
+
+      if (!characterData) {
+        server.sendChatText(client, `Character data not found.`);
+        return;
+      }
+
+      const stats = {
+        "Zombie Kills": characterData.stats?.zombiesKilled ?? 0,
+        "Playtime": `${characterData.playTime ?? 0} hours`,
+        "PvP Kills": characterData.stats?.playersKilled ?? 0,
```

Comment:
players/zombie kills and death are already stored in the mongo collection "kills"